File: DotNetEmploymentTimes/DotNetEmploymentTimes/Utils/MySQLConnection.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using MySql.Data.MySqlClient;

namespace DotNetEmploymentTimes.Utils
{
    public  class MySQLEMPConnection
    {
        static MySqlConnection con = null;
        public static MySqlConnection GetConnection()
        {
            try
            {
                if (con == null)
                {
                     con = new MySqlConnection("SERVER=localhost;DATABASE=EMPDB;UID=root;PASSWORD=sa;");
                }
                else
                {
                    return con;
                }

            }
            catch (Exception e)
            {
                throw;
            }

            return con;
        }

    }
}

File: JavaEmploymentTimes1/src/main/java/com/JavaEmploymentTimes/Util/MySQLConnection.java
package com.JavaEmploymentTimes.Util;

import java.sql.Connection;
import java.sql.DriverManager;

public class MySQLConnection {


	private static Connection con = null;

	public static Connection GetConnection() {
		try {

			if (con == null) {
				String driver = "com.mysql.jdbc.Driver";
				String URL = "jdbc:mysql://localhost:3306/empdb";
				String user = "root";
				String password = "root";

				Class.forName(driver);
				con = DriverManager.getConnection(URL, user, password);
			}
		} catch (Exception e) {

			e.printStackTrace();
		}

		return con;
	}

}
